pkg/config/config.go

```diff
@@ -66,10 +66,15 @@ func NewState(configPath string, servicePath string) (*State, error) {
 		return nil, fmt.Errorf("Service Directory Missing")
 	}
 
-	if _, err := os.Stat(servicePath); os.IsNotExist(err) {
+	var mode os.FileInfo
+	if mode, err = os.Stat(servicePath); os.IsNotExist(err) {
 		return nil, fmt.Errorf("%v: %s", err, servicePath)
 	}
 
+	if !mode.IsDir() {
```

Comment:
this can be combined with the above in an else if, so we don't need the mode floating around.